convokit/model/corpus.py

```diff
@@ -33,124 +41,227 @@ class Corpus:
         Type-checking ensures that the ConvoKitIndex is initialized correctly. However, it may be unnecessary if the
         index.json is already accurate and disabling it will allow for a faster corpus load. This parameter is set to
         True by default, i.e. type-checking is not carried out.
+    :param storage_type: either 'mem' or 'db'
```

Comment:
This is not a required argument, so maybe rewrite this as: optionally specify the storage_type ('mem', 'db'), leave as None to use default...etc